ci-operator/step-registry/layered-product/acm/orchestrate/crd-application/layered-product-acm-orchestrate-crd-application-commands.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -x
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "This is the interop-tooling-crd-application-command.sh space running"
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+echo "Apply multiclusterhub"
+# apply MultiClusterHub crd
+oc apply -f - <<EOF
+apiVersion: operator.open-cluster-management.io/v1
+kind: MultiClusterHub
+metadata:
+  name: multiclusterhub
+  namespace: open-cluster-management
+spec: {}
+EOF
+
+# can't wait before the resource exists. Need to sleep a bit before start watching
+sleep 60
+
+RETRIES=30
+for i in $(seq "${RETRIES}"); do
+  if [[ $(oc get mch -n open-cluster-management -o=jsonpath='{.items[0].status.phase}') == "Running" ]]; then
+    echo "MCH is Running"
+    break
+  else
+    echo "Try ${i}/${RETRIES}: MCH is not running yet. Checking again in 30 seconds"
+    sleep 30
```

Comment:
max cluster is 30?!

Reply:
This is just saying that it will retry 30 times to check to see if oc get mch is returning "Running".

I'll sleep for 30 seconds inbetween retries.

So this loop will be active for up to 15 minutes waiting for mch to become running